t/80http2-stream-limit.t

```diff
@@ -0,0 +1,99 @@
+use strict;
```

Comment:
Please rename the file to t/40http2-stream-limit.t, as 40 is the group we use for testing protocol implementations (80 are tests related to specific bugs).

Reply:
Done.